src/index.html

```diff
@@ -1,12 +1,409 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <title>Catalog</title>
```

Comment:
I'm pretty sure we have font-weigth 700 also, but you choose only 400, 500. I'm not sure but check if its change something..font-weight 700 we will use in price $23233434